receiver/redisreceiver/metrics_service.go

```diff
@@ -0,0 +1,37 @@
+package redisreceiver
+
+import (
+	resourceProto "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
```

Comment:
For uniformity with the rest of the codebase:
```suggestion
	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
```

Reply:
Fixed.